spark/src/main/scala/org/apache/comet/rules/CometExecRule.scala

```diff
@@ -183,21 +183,6 @@ case class CometExecRule(session: SparkSession) extends Rule[SparkPlan] {
    */
   // spotless:on
   private def transform(plan: SparkPlan): SparkPlan = {
-    def operator2ProtoIfAllChildrenAreNative(op: SparkPlan): Option[Operator] = {
-      if (op.children.forall(_.isInstanceOf[CometNativeExec])) {
-        operator2Proto(op, op.children.map(_.asInstanceOf[CometNativeExec].nativeOp): _*)
-      } else {
-        None
-      }
-    }
-
-    /**
-     * Convert operator to proto and then apply a transformation to wrap the proto in a new plan.
-     */
-    def newPlanWithProto(op: SparkPlan, fun: Operator => SparkPlan): SparkPlan = {
-      operator2ProtoIfAllChildrenAreNative(op).map(fun).getOrElse(op)
-    }
```

Comment:
I moved these methods out of the `transform` method so that they can be accessed elsewhere